NEWS.md

```diff
@@ -39,6 +39,10 @@
    Because there was no recorded direct usage for any of these, I am opting to just rip the band-aid
    off and un-export them in this release as opposed to waiting a full cycle more to do so.
 
+## BUG FIXES
+
+1. `min.integer64`, `max.integer64` and `range.integer64` now support `na.rm=TRUE` correctly (#142).
```

Comment:
please add a simple example here for readers to get a sense of what used to be wrong without needing to dig into details

Reply:
Do you think that summarizes the issue well?

```suggestion
1. `min.integer64`, `max.integer64` and `range.integer64` now support `na.rm=TRUE` correctly  when combining across mutliple inputs like `min(x, NA_integer64_, na.rm=TRUE)` (#142).
```